CRM/Core/Payment/Form.php

```diff
@@ -319,7 +319,7 @@ public static function validateCreditCard($values, &$errors, $processorID = NULL
       if (!empty($values['credit_card_type'])) {
         $processorCards = CRM_Financial_BAO_PaymentProcessor::getCreditCards($processorID);
         if (!empty($processorCards) && !in_array($values['credit_card_type'], $processorCards)) {
-          $errors['credit_card_type'] = ts('This procesor does not support credit card type ' . $values['credit_card_type']);
+          $errors['credit_card_type'] = ts('This procesor does not support credit card type %1', [1 => $values['credit_card_type']]);
```

Comment:
@bgm As we are touching this string can we correct the spelling: procesor => processor

Reply:
@mattwire Good catch, I updated my PR.